momepy/functional/_distribution.py

```diff
@@ -339,3 +343,79 @@ def street_alignment(
     Series
     """
     return (building_orientation - street_orientation.loc[street_index].values).abs()
+
+
+def describe(
+    values: NDArray[np.float_] | Series,
```

Comment:
Seems to me like many of the mainstream graph-based packages generally have graph-like object as the first positional argument. How would we feel about swapping `values` & `graph` here?

Reply:
So far, we've been using this order. In alignment we have `(orientation, graph)`, in neighbor_distance we have `(geometry, graph)`, mean_interbuilding_distance has `(geometry, adjacency_graph, neighborhood_graph)`. I don't mind much in which order it is gonna be as long as we can make it consistent.

What we have here more mirrors what we have elsewhere in pysal, like esda's `Geary(y, w)`. (which also reminds me that we should probably rename `values` to `y` or `a` or something that is common in the ecosystem.

Reply:
> So far, we've been using this order

Let's keep the order how it is then.